p/PLSQL.sql

```diff
@@ -1,4 +0,0 @@
-begin
-  dbms_output.put_line("Hello World");
-end;
-/
```

Comment:
What is the purpose of this; should this be transferred to the new file?

Reply:
I deleted it because
* the file extension for PL/SQL is .pls
* it should use single quotes not double quotes